### PR TITLE
New package: f3-6.0

### PR DESCRIPTION
**Pull request by bra1nwave**

### CODE REVIEW

File: srcpkgs/f3/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version=6.0
+revision=1
+build_style=gnu-makefile
+short_desc="Test real flash memory capacity"
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="GPL-3"
+homepage="https://github.com/AltraMayor/f3"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=d72addb15809bc6229a08ac57e2b87b34eac80346384560ba1f16dae03fbebd5
+nopie=yes
```
Comment (Vaelatern) [inline srcpkgs/f3/template]: Why?

Comment (bra1nwave) [inline srcpkgs/f3/template]: I get this error, even when I provide $LDFLAGS:
> [...]
=> f3-6.0_1: running post-install hook: 06-strip-and-debug-pkgs ...
   Stripped executable: /usr/bin/f3read
=> ERROR: f3-6.0_1: non-PIE executable found in PIE build: /usr/bin/f3read
=> ERROR: f3-6.0_1: failed to run post-install_06-strip-and-debug-pkgs() at line 58.`

Comment (Vaelatern) [inline srcpkgs/f3/template]: So where are our LDFLAGS and CFLAGS being ignored?

**Comment by bra1nwave** [comment]
I think i found the Error. Thanks!

File: srcpkgs/f3/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="Test real flash memory capacity"
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="GPL-3"
+homepage="https://github.com/AltraMayor/f3"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=d72addb15809bc6229a08ac57e2b87b34eac80346384560ba1f16dae03fbebd5
+
+do_build() {
+	sed -i 's/^\(CFLAGS *+=\)/override \1/' Makefile
+	make CFLAGS="$CFLAGS $LDFLAGS"
```
Comment (Gottox) [inline srcpkgs/f3/template]: why not fixing both CFLAGS and LDFLAGS in the patch above?

Comment (bra1nwave) [inline srcpkgs/f3/template]: I hope i've understood you correctly and it's ok now. If not, can you give me hint please?

**Pull request merged**